source/css/components/_arrow.scss

```
@@ -0,0 +1,44 @@
+// To be used with partials/components/_arrow.html.erb
+svg.arrow {
```

Super minor - but is the specificity of the tag (`svg`) needed?

---

No.  It is less performant, but to me here it helped describe it a bit better and make it more safe?  Does it _add anything_ to you?  If not:
```suggestion
.arrow {
```

---

Not too worried either way here - it does make it a bit safer and more understandable, even though I usually try to keep specificity low 😄